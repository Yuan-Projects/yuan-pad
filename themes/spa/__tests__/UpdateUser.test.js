import renderer from 'react-test-renderer';
import UpdateUser from '../src/updateUser';
import AdminUser from '../__mocks__/user-admin';
import RegularUser from '../__mocks__/user-regular';
import LangEn from '../__mocks__/site-translate-en';

it('UpdateUser renders correctly', () => {
  let props1 = {
    user: AdminUser,
    lang: LangEn,
    onCurrentUserUpdated: () =>{}
  };
  const component1 = renderer.create(<UpdateUser
    {...props1}
  />);
  let tree1 = component1.toJSON();
  expect(tree1).toMatchSnapshot();

  let props2 = {
    user: RegularUser,
    lang: LangEn,
    onCurrentUserUpdated: () =>{}
  };
  const component2 = renderer.create(<UpdateUser
    {...props2}
  />);
  let tree2 = component2.toJSON();
  expect(tree2).toMatchSnapshot();
});