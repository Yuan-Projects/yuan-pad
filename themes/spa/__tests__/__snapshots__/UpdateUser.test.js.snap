// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpdateUser renders correctly 1`] = `
<div
  className="updateUser"
>
  <a
    className="btn btn-default"
    href="#"
    onClick={[Function]}
    role="button"
  >
    Update
  </a>
</div>
`;

exports[`UpdateUser renders correctly 2`] = `
<div
  className="updateUser"
>
  <a
    className="btn btn-default"
    href="#"
    onClick={[Function]}
    role="button"
  >
    Update
  </a>
</div>
`;
